documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -83,20 +83,61 @@ Typically, the redo logs on an Oracle server are configured to not retain the co
 
-1. Determines the tables to be captured.
-2. Obtains a `ROW SHARE MODE` lock on each of the captured tables to prevent structural changes from occurring during creation of the snapshot. {prodname} holds the locks for only a short time.
-3. Reads the current system change number (SCN) position from the server's redo log.
-4. Captures the structure of all relevant tables.
-5. Releases the locks obtained in Step 2.
-6. Scans all of the relevant database tables and schemas as valid at the SCN position that was read in Step 3 (`SELECT * FROM ... AS OF SCN 123`), generates a `READ` event for each row, and then writes the event records to the table-specific Kafka topic.
-7. Records the successful completion of the snapshot in the connector offsets.
+1. Establish a connection to the database.
+
+2. Determine the tables to be captured.
+By default, the connector captures all non-system tables.
+To have the connector capture a subset of tables or table elements, you can set a number of `include` and `exclude` properties to filter the data, for example, xref:oracle-property-table-include-list[`table.include.list`] or xref:oracle-property-table-exclude-list[`table.exclude.list`].
+
+3. Obtains a `ROW SHARE MODE` lock on each of the captured tables to prevent structural changes from occurring during creation of the snapshot.
+   {prodname} holds the locks for only a short time.
+
+4. Reads the current system change number (SCN) position from the server's redo log.
+
+5. Captures the structure of all database tables.
+The connector persists this information in its internal database schema history topic.
+The schema history provides information about the structure that is in effect when a change event occurs. +
++
+[NOTE]
+====
+By default, the connector captures the schema of every table in the database that is in capture mode, including tables that are not configured for capture.
+However, for tables that are not configured for capture, the initial snapshot captures only their structure; it does not capture any table data.
+
+There is a benefit to persisting schema information for tables that you did not include in the initial snapshot.
+When {prodname} has information about the structure of all existing tables, it can readily capture events from any existing table should you later decide that is needed.
+If you're certain that it won't be necessary to capture data from other tables in the future, you can modify the default behavior by changing the value of the xref:{context}-property-database-history-store-only-captured-tables-ddl[`schema.history.internal.store.only.captured.tables.ddl`] property to `true`.
+====
+
+6. Releases the locks obtained in Step 3.
+Other database clients can now write to the database.
+
+7. At the SCN position that was read in Step 4, the connectors scans the tables that are designated for capture (`SELECT * FROM ... AS OF SCN 123`).
+During the scan, the connector completes the following tasks:
```

Comment:
I wonder if we shouldn't consider adding a small note about `UNDO_RETENTION` and the snapshot duration and the potential for ORA-01555; see the Oracle FAQ at the end about "What’s the cause for ORA-01555 and how to handle it?".